play/PlayerRules/CharacterCreation/Class-Specific Documentation/Power Suits/Power Suit Rules.txt

```diff
@@ -0,0 +1,37 @@
+		Overview
+	Some militaries in the Terran Empire have employed the use of 
+large mechanical warsuits operated by a single pilot. This class of
+armor-vehicle hybrid is colloquially called "Power Suits." Power 
+Suits tend to be slow, tough and pack an ammount of punch rarely 
+seen by infantry. 
+
+		Power
+	As the name implies, most power suits carry a power plant
+on them that runs their various weapons, parts and chassis. A
+Power Suit may carry, but not activate or use a part or weapon
+that would exceed the power capacity of the suit's power plant.
+
+		Modes
+	Power Armor has three modes. Switching between modes takes
+Two actions for a pilot in the armor. These modes are:
+		- Combat
+		- Maneuver
+		- Shutdown
+	In combat mode, the suit works as expected. The pilot
+may direct the armor to move, fire and scan using its various 
+actions.
+	In Maneuver mode, the suit cannot use any of its active
+components or weapons. However it gains a bonus to move speed as
+listed in the chassis's Maneuver Bonus. 
+	In Shutdown mode, the suit is much more difficult to hack.
+However the suit cannot scan, move nor use active components or 
+weapons. If the Power Suit's power output no longer meets or 
+exceeds its power requirement, it immediately enters shutdown mode.
+
+		Cover
+	Power Suits are much larger than standard infantry, and 
+are typically a little less graceful to boot. They Cannot use the
+hunker action. Furthermore partial cover has no effect on the 
+accuracy of an enemy firing at a Power Suit and good cover acts 
+as parital cover for power suits. Due to their sheer size, enemies
+get a flat -20% to miss when firing at power suits. 
```

Comment:
Unless I misunderstand your wording... I don't think it's necessary to double stack penalties against the power suit: I'd give the suits -20% to miss against __OR__ the cover nerfs (and no hunkering in either case). As far as a I can tell In terms of realism, the -20% miss chance and cover nerfs are the same physical concept being penalized, large size = easier to hit so we can probably get away with 1 concept vs 2 to lessen the complexity. If that makes sense.

If you agree, it might be good to use the -20% miss chance. It's less intuitive, imo, but it allows the suit to be hit better in no cover, causes light cover be of no normal effect and heavy to be ~20% . 

By the way, can power armor crouch or go prone?